main.tex

```diff
@@ -58,15 +58,14 @@
 \begin{abstract}
     Memory-one strategies are a set of Iterated Prisoner's Dilemma strategies
     that have been praised for their mathematical tractability and performance
-    against single opponents. This manuscript investigates \textit{best
-    response} memory-one strategies as a multidimensional
-    optimisation problem. Though extortionate memory-one strategies have gained
-    much attention, we demonstrate that best response memory-one strategies do not
-    behave in an extortionate way, and moreover, for memory-one strategies to be
-    a best response to a group of opponents and themselves they need to be able to behave in a forgiving way. We
+    against single opponents. This manuscript investigates a theory of mind: \textit{best
+    response} memory-one strategies, as a multidimensional
+    optimisation problem. We add to the literature that has shown that
+    extortionate play is not always optimal by showing that optimal play is in
+    fact rarely extortionate. We
```

Comment:
I am not sure about the word rarely. The distribution of the SSE is skewed towards zero.

Reply:
Yeah `rarely` is an exaggeration. Perhaps `that optimal play is often not extortionate`?

Reply:
Yup that sounds better 👍